src/main/java/com/lob/net/ResponseGetter.java

```diff
@@ -92,7 +92,7 @@ public Parameter(String key, Object value) {
     private static java.net.HttpURLConnection createDefaultConnection(String url, RequestOptions options) throws IOException {
         URL lobURL = new URL(url);
 
-        HttpURLConnection conn = (HttpURLConnection) lobURL.openConnection();
+        HttpURLConnection conn = (HttpURLConnection) (options.getProxy() == null ? lobURL.openConnection() : lobURL.openConnection(options.getProxy()));
```

Comment:
Nit: Could you flip the conditional here to check for existence instead of non-existence?

```suggestion
        HttpURLConnection conn = (HttpURLConnection) (options.getProxy() ? lobURL.openConnection(options.getProxy()) : lobURL.openConnection());
```